src/main/java/graphql/GraphQL.java

```diff
@@ -66,7 +119,7 @@ public ExecutionResult execute(String requestString, String operationName, Objec
             RecognitionException recognitionException = (RecognitionException) e.getCause();
             SourceLocation sourceLocation = new SourceLocation(recognitionException.getOffendingToken().getLine(), recognitionException.getOffendingToken().getCharPositionInLine());
             InvalidSyntaxError invalidSyntaxError = new InvalidSyntaxError(sourceLocation);
-            return new ExecutionResultImpl(Arrays.asList(invalidSyntaxError));
+            return new ExecutionResultImpl(Collections.singletonList(invalidSyntaxError));
```

Comment:
Smidge more efficient and IDEA inspections (which I trust mostly) says to do it. 

Broken window theory in code and all...